docker-compose.yml

```diff
@@ -50,7 +55,7 @@ services:
   #  
   rabbitmq:
     container_name: "${SHANOIR_PREFIX}rabbitmq"
-    image: rabbitmq:3.7.7-management-alpine
```

Comment:
this change seems unrelated to keycloak

Reply:
Hi Anthony, I agree for the general point of view, but Shanoir does not work on my machine without that update and as it works as well on the other machines, I am happy to bring it into develop anyway. I will add it into the description, that we are not surprised if we might have side effects. As I tested imports on my machine, there should not be one except updating rabbitmq and using a newer version.